android/cli/commands/_build.js

```diff
@@ -2658,7 +2658,7 @@ AndroidBuilder.prototype.copyResources = function copyResources(next) {
 
 			// encrypt the javascript
 			var titaniumPrepHook = this.cli.createHook('build.android.titaniumprep', this, function (exe, args, opts, done) {
-					this.logger.info(__('Encrypting JavaScript files: %s', (exe + ' "' + args.join('" "') + '"').cyan));
+					this.logger.info(__('Encrypting JavaScript files: %s', (exe + ' "' + args.slice(0, -1).join('" "') + '"').cyan));
```

Comment:
This might not be applicable for Android platform. The guid is always the beginning of the args.